test/multiarch_utils.go

```diff
@@ -46,6 +47,7 @@ func getTestArch() string {
 	return runtime.GOARCH
 }
 
+// return the map with arch dependent image names for e2e tests
```

Comment:
Go style nit: Doc comments should be in the form of `// methodName returns the map ...` -- in other words, the first word in the comment should be the method name.

Reply:
done